menpo/image/base.py

```diff
@@ -1175,8 +1175,8 @@ def constrain_points_to_bounds(self, points):
         bounded_points[over_image] = shape[over_image]
         return bounded_points
 
-    def extract_patches(self, patch_centers, patch_size=(16, 16),
-                        sample_offsets=None, as_single_array=False):
+    def extract_patches(self, patch_centers, patch_shape=(16, 16),
```

Comment:
Following our discussion with @patricksnape and @jabooth , it is agreed that we always use `patch_shape` and we make it consistent throughout menpo and menpofit.